### PR TITLE
[Feat] Support Policy page

### PR DESCRIPTION
**Pull request by macrigiuseppe**
Created new support policy page

Signed-off-by: Giuseppe Macri <gmacri@uber.com>

### CODE REVIEW

File: website/src/components/policy.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+const GITHUB_PROJECT_ISSUES = `${GITHUB_PROJECT}/issues`;
+
+const StyledContainer = styled(Container)`
+  background-color: black;
+  height: 100vh;
+  
+  max-width: unset;
+  
+  .kg-home__caption__description {
+    max-width: unset;
```
Comment (heshan0131) [inline website/src/components/policy.js]: can you add a max-width: 800px to this, so it doesn't expand all the way to a 2000px widescreen?

File: website/src/components/policy.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              <StyledCaption>
+                <div className="kg-home__caption__subtitle">
+                  Support Policy
+                </div>
+                <div className="kg-home__caption__description">
+                  <h3>
+                    <a href="https://kepler.gl">Kepler.gl</a> is an open source project. Its source code is available at <a
+                    href={GITHUB_PROJECT}>Kepler.gl Github project</a>.
+                  </h3>
+                  <span>
```
Comment (heshan0131) [inline website/src/components/policy.js]: make this `font-weight: 500`?

File: website/src/components/policy.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                  As an open source project, questions and bug reports can be filed with the project community,
+                  and you can participate in the development of the project's source code. Bug reports and
+                  issues can be submitted at <a
+                    href={GITHUB_PROJECT_ISSUES}>Kepler.gl Github issues</a>.
+                  [Questions can be asked on the project's mailing lists, which you can subscribe to at [URL] // NEED TO BE DEFINED]
```
Comment (macrigiuseppe) [inline website/src/components/policy.js]: @heshan0131 can you help with this? What should be the url for mailing list subscription? if we have one

Comment (heshan0131) [reply website/src/components/policy.js]: created:
https://groups.google.com/d/forum/kepler-gl

**Pull request merged**